libbeat/scripts/Makefile

```diff
@@ -101,7 +101,7 @@ crosscompile: $(GOFILES)
 check: python-env ## @build Checks project and source code if everything is according to standard
 	@go vet ${GOPACKAGES}
 	@go get $(GOIMPORTS_REPO)
-	@goimports -local ${GOIMPORTS_LOCAL_PREFIX} -l ${GOFILES_NOVENDOR} | (! grep . -q) || (echo "Code differs from goimports' style" && false)
```

Comment:
No reason. My main goal was to get a non-zero exit code. I expected that if it failed you would run `goimports` to list the files and fix the issue.